client/src/pages/Home/MemberSays/Member.css

```diff
@@ -0,0 +1,36 @@
+.grid-container {
+	display: grid;
+	grid-template-columns: auto auto auto;
+	padding-right: 60px;
+	padding-left: 60px;
+	padding-bottom: 10px;
+	column-gap: 60px;
+}
+.grid-item {
+	background-color: white;
+	border: 1px solid rgba(0, 0, 0, 0.8);
+	padding: 20px;
+	font-size: 30px;
+	text-align: center;
+}
+.button:hover {
+	box-shadow: 0 12px 16px 0 rgba(0, 0, 0, 0.24),
+		0 17px 50px 0 rgba(0, 0, 0, 0.19);
+	background-color: grey;
+	color: black;
+}
+.button {
```

Comment:
can you improve the class name i think button is not a good class name.

Reply:
Changed to register-btn. Done